rosdep/python.yaml

```diff
@@ -5494,6 +5494,13 @@ python-watchdog:
     xenial: [python-watchdog]
     yakkety: [python-watchdog]
     zesty: [python-watchdog]
+python-weasyprint-pip:
```

Comment:
do you want to add an entry for python3 or python2?
For python3 please use `python3-weasyprint-pip`.

The package seems to only be available for python3.

Reply:
changed